lib/routes/admin-api/feature.js

```diff
@@ -115,8 +115,8 @@ module.exports.router = function(config) {
     router.post('/', (req, res) => {
         req.checkBody('name', 'Name is required').notEmpty();
         req
-            .checkBody('name', 'Name must match format ^[0-9a-zA-Z\\.\\-\\_]+$')
-            .matches(/^[0-9a-zA-Z\\.\\-\\_]+$/i);
+            .checkBody('name', 'Name must match format ^[0-9a-zA-Z\.\-\_]+$')
+            .matches(/^[0-9a-zA-Z\.\-\_]+$/i);
```

Comment:
`/i` makes no sense

Reply:
Also, just extract it into a constant so we don't have to duplicate it (you can run `.source` on a regex to get the underlying string out)

Reply:
I agree, will fix. The regex can be simplified to `/^[0-9a-zA-Z\-._]+$/`